docs/src/environments/_index.md

```diff
@@ -1,118 +1,165 @@
+If you use a [source integration](../integrations/source/_index.md),
+you can even have environments created automatically for your pull requests and branches.
+
+You can also have branches that aren't tied to a running instance of your application.
+These are called [inactive environments](#environment-status).
 
 ## Default environment
 
-Your default environment comes from your default branch and is a [Production environment](../administration/users.md#environment-types).
+Your default environment comes from your default branch and is a [production environment](../administration/users.md#environment-types).
+Your project must have a default environment,
+but you can [name it as you want](./default-environment.md).
+
+If you subscribed to a production plan, this environment is your **live site**.
+You might want to give it a [custom domain name](../domains/steps/_index.md).
+
+## Environment status
+
+Your environments can have one of two statuses:
+
+-   [Active](../other/glossary.md#active-environment):
+    A deployed environment with services and data.
+
+-   [Inactive](../other/glossary.md#inactive-environment):
+    An environment that isn't deployed and has no services or data, only code.
+
+You can see the status of your environments in the [Console](../administration/web/_index.md) or the [CLI](/administration/cli/_index.md).
+
+{{< codetabs >}}
++++
+title=In the Console
++++
+
+When you open your project, inactive environments appear lighter in the environment list.
+You also can't select them from the **Environment** dropdown menu.
+
+To check the status of an environment,
+you can also open it and view its information panel.
 
-If you subscribed to a production plan, this environment is your **live site**
-and you can give it a [custom domain name](../domains/steps/_index.md) and a custom SSL certificate.
+<--->
++++
+title=Using the CLI
++++
 
-{{< note >}}
+To check the status of all your environments, from your project directory run the `platform environments` command.
 
-Your project must have a default environment, but you can give it any name.
-See how to [rename the default environment](./default-environment.md).
+{{< /codetabs >}}
 
-{{< /note >}}
+You can [change an environment's status](./deactivate-environment.md) at any time.
 
-## Hierarchy
+## Organize your environments
+
+### Hierarchy
 
 ![Environment hierarchy](/images/management-console/environments.png "0.5")
 
-In Platform.sh, your environments are organized in a hierarchy.
-Each new environment you create is considered a **child** of the **parent** environment from which it was created.
+In Platform.sh, your environments are organized in a hierarchy featuring parent and child environments.
+
+When you [branch](../other/glossary.md#branch) an environment,
+the parent of the new environment is the environment it was created from.
+You can [change the environment's parent](./change-parent.md) after it's been created using the [CLI](/administration/cli/_index.md).
```

Comment:
:hammer_and_wrench: This makes it seem like it's only possible with the CLI, while the link shows how to do it in the Console.
```suggestion
You can [change the environment's parent](./change-parent.md) after it's been created.
```